README.md

```diff
@@ -14,6 +14,26 @@ it, simply add the following line to your `Podfile`:
 pod "Segment-Appboy"
 ```
 
+## Usage
+
+After adding the dependency, you must register the integration with our SDK. To do this, import the Appboy integration in your AppDelegate:
+
+
+```
+#import <Segment-Appboy/AppboyIntegrationFactory.h>
```

Comment:
I tested with the import statement that you have here, and the project didn't build. Would you be able to either replace it with the import statement we use in our sample app (`#import "SEGAppboyIntegrationFactory.h"`) or share your Podfile and integration method so we can see how you're integrating?